bundle

```diff
@@ -48,14 +49,21 @@ def get_args():
                         help='the bundle fragments to include in this bundle')
     return parser.parse_args()
 
-
-def fragment_paths(fragment):
+def fragment_paths(fragment, converged=False):
     ''' Get the root, bundle, and readme paths for a fragment. '''
     frag_path = os.path.join(root_path, 'fragments',
                              os.path.join(*fragment.split('/')))
+
+    bundle_path = os.path.join(frag_path, 'bundle.yaml')
+
+    # If a converged file exists, include it otherwise use default
+    if converged:
+       if os.path.isfile(os.path.join(frag_path,'bundle-converged.yaml')):
```

Comment:
Indent should be 4 spaces, this looks like 3? Same for line below, appears under-indented.